mars-app-spa/src/FCs/dropDownFC.tsx

```diff
@@ -1,9 +1,17 @@
-import React, { useContext, useEffect, useState, Component  } from 'react';
-import Select from 'react-select'
+import React from "react";
+import Select from "react-select";
+import "./dropDownFC.css";
 
-export const DropDownFC:React.FC <{options:{value:string,label:string}[]}> = ({options}) => {
-    return (
-        <Select options={options}
-                onChange = {(x) => console.log(x)}/>
-    )
-}
\ No newline at end of file
+export const DropDownFC: React.FC<{

```

Comment:
You could do with some explicit type definitions here - a common pattern is to define a `Props` type immediately before the component which uses them - something like
```
type DropDownFCProps = {
  options: Option[];
  onSelect: ...
}
}
```